pkg/controller/aws/provider/provider.go

```diff
@@ -112,7 +113,7 @@ func (r *Reconciler) _validate(config *aws.Config) error {
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=aws.crossplane.io,resources=provider,verbs=get;list;watch;create;update;patch;delete
 func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	log.Printf("reconciling %s: %v", awsv1alpha1.ProviderKindAPIVersion, request)
+	logger.V(1).Info("reconciling", "kind", awsv1alpha1.ProviderKindAPIVersion, "request", request)
```

Comment:
Here and everywhere, would you consider a slightly different symantics:
```go
import logx "github.com/crossplaneio/crossplane/pkg/log"
...
log = logx.Log.WithName("controller" + controllerName)
log.Info("...")
// or
log.V(1).Info("...")
```

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md

I chose this naming based on the above logging practices guideline, but I also do not like the stuttering name. In general I've tried to use the same patterns as the controller runtime despite not personally liking any of the controller-runtime logging patterns very much.

If I were to tweak the naming myself, I would rename our `log` package to `logging` and rename `logging.Log` to `logging.Logger`. This would have our code read like:

```go
import "github.com/crossplaneio/crossplane/pkg/logging

var log = logging.Logger.WithName("controller." + controllerName)

func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
  log.V(1).Info("...")
}
```

Reply:
I implemented the above renaming in a new commit.